testsuite/features/reposync/srv_enable_sync_products.feature

```diff
@@ -8,35 +8,10 @@ Feature: Be able to list available products and enable them
 
   Scenario: List available products
     When I execute mgr-sync "list products" with user "admin" and password "admin"
-    Then I should get "[I] SUSE Linux Enterprise Server 12 SP5 x86_64"
-    And I should get "[ ] SUSE Linux Enterprise Desktop 15 SP3 x86_64"
+    Then I should get "[ ] SUSE Linux Enterprise Desktop 15 SP3 x86_64"
 
   Scenario: List all available products
     When I execute mgr-sync "list products -e"
     Then I should get "[ ] SUSE Linux Enterprise Desktop 15 SP3 x86_64"
     And I should get "  [ ] (R) Basesystem Module 15 SP3 x86_64"
     And I should get "  [ ] Desktop Applications Module 15 SP3 x86_64"
-
-  Scenario: Enable "SUSE Linux Enterprise Desktop 15 SP3 x86_64" with recommended modules
-    # do NOT use same SP as for SLES minions
-    # ask mcalmer for technical details why this is a problem
-    When I enable product "SUSE Linux Enterprise Desktop 15 SP3 x86_64"
-    Then I should get "Adding channels required by 'SUSE Linux Enterprise Desktop 15 SP3 x86_64' product"
-    And I should get "- sle-product-sled15-sp3-updates-x86_64"
-    And I should get "- sle-product-sled15-sp3-pool-x86_64"
-    And I should get "- sle-module-basesystem15-sp3-updates-x86_64-sled"
-    And I should get "- sle-module-basesystem15-sp3-pool-x86_64-sled"
-    And I should get "- sle-module-desktop-applications15-sp3-updates-x86_64-sled"
-    And I should get "- sle-module-desktop-applications15-sp3-pool-x86_64-sled"
-    And I should get "- sle-product-we15-sp3-updates-x86_64-sled"
-    And I should get "- sle-product-we15-sp3-pool-x86_64-sled"
-    And I should get "Product successfully added"
```

Comment:
Just curious, why do you remove this part? You only mention SLES 12 in the PR description. I guess we only use SLES 15 SP4 in the CI!?

Reply:
I did forget to mention sled15 sp3, however similarly we start syncingthe product and never use it afterwards
I'll add it to the description

Reply:
Ah, I just saw that this is SLED noch SLES. Sorry for the noise.

Reply:
Just did a test run while inducing the error, the `log` does seem to be working:
![image](https://user-images.githubusercontent.com/100688791/222476225-57e28c20-79ed-40df-8eb5-c0eab53396fe.png)